bitnami/redis/templates/redis-master-statefulset.yaml

```diff
@@ -217,6 +217,8 @@ spec:
             - name: REDIS_ALIAS
               value: {{ template "redis.fullname" . }}
             {{- if and .Values.usePassword (not .Values.usePasswordFile) }}
+            - name: REDIS_USER
+              value: default
```

Comment:
In the `redis-slave` and `redis-node` statefulset, this is above the IF statement, but in this one, you added it below.